flake8 issues
```
./components/tools/OmeroPy/test/integration/clitest/test_cleanse.py:40:80: E501 line too long (83 > 79 characters)
./components/tools/OmeroPy/test/integration/clitest/test_cleanse.py:47:1: E302 expected 2 blank lines, found 1
./components/tools/OmeroPy/test/integration/clitest/test_cleanse.py:56:80: E501 line too long (83 > 79 characters)
./components/tools/OmeroPy/test/integration/clitest/test_cleanse.py:60:80: E501 line too long (98 > 79 characters)
```

--- a/components/tools/OmeroPy/test/integration/clitest/test_cleanse.py
+++ b/components/tools/OmeroPy/test/integration/clitest/test_cleanse.py
@@ -62,5 +62,5 @@ class TestCleanseRoot(RootCLITest):
         out, err = capsys.readouterr()
         output_string_start = "Removing empty directories from...\n "
         mrepo_dir = config_service.getConfigValue("omero.managed.dir")
-        output_string = mrepo_dir.replace("//", "/")
+        output_string = output_string_start + mrepo_dir.replace("//", "/")
         assert output_string in out

--- a/components/tools/OmeroPy/test/integration/clitest/test_cleanse.py
+++ b/components/tools/OmeroPy/test/integration/clitest/test_cleanse.py
@@ -37,12 +37,14 @@ class TestCleanse(CLITest):
 
     def testCleanseAdminOnly(self, capsys):
         """Test cleanse is admin-only"""
-        data_dir = self.root.sf.getConfigService().getConfigValue("omero.data.dir")
+        config_service = self.root.sf.getConfigService()
+        data_dir = config_service.getConfigValue("omero.data.dir")
         self.args += [data_dir]
         with pytest.raises(NonZeroReturnCode):
             self.cli.invoke(self.args, strict=True)
         out, err = capsys.readouterr()
         assert err.endswith("SecurityViolation: Admins only!\n")
+
 
 class TestCleanseRoot(RootCLITest):
 
@@ -53,9 +55,11 @@ class TestCleanseRoot(RootCLITest):
 
     def testCleanseAdminOnly(self, capsys):
         """Test cleanse works for root with expected output"""
-        data_dir = self.root.sf.getConfigService().getConfigValue("omero.data.dir")
+        config_service = self.root.sf.getConfigService()
+        data_dir = config_service.getConfigValue("omero.data.dir")
         self.args += [data_dir]
         self.cli.invoke(self.args, strict=True)
         out, err = capsys.readouterr()
-        output_string = "Removing empty directories from...\n " + data_dir + "ManagedRepository\n"
+        output_string_start = "Removing empty directories from...\n "
+        output_string = output_string_start + data_dir + "ManagedRepository\n"
         assert output_string in out

--- a/components/tools/OmeroPy/test/integration/clitest/test_cleanse.py
+++ b/components/tools/OmeroPy/test/integration/clitest/test_cleanse.py
@@ -61,5 +61,6 @@ class TestCleanseRoot(RootCLITest):
         self.cli.invoke(self.args, strict=True)
         out, err = capsys.readouterr()
         output_string_start = "Removing empty directories from...\n "
-        output_string = output_string_start + data_dir + "ManagedRepository\n"
+        mrepo_dir = config_service.getConfigValue("omero.managed.dir")
+        output_string = mrepo_dir.replace("//", "/")
         assert output_string in out

